roles/opnsense_configure/README.md

```diff
@@ -0,0 +1,116 @@
+opnsense_configure - OPNsense configuration role
+=========
+
+This role provides a generic approach to configure OPNsense instances by populating host variables
+according to this roles defaults specification.
+
+Role Variables
+--------------
+
+The variables must be structured in a way that each puzzle.opnsense module has its own block. Each module related block
```

Comment:
Does Block make sense in this context? I, for one, thought about an Ansible Block, which was kind a confusing... maybe `dictionary` or `map` are better words than `block` :thinking:

Reply:
You are right, this can be confusing. I opted for `variable section` to replace `block`.